File: src/C#/BehavioralPatterns/Command/FastFoodRestaurant/Models/MenuItem.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FastFoodRestaurant.Models
{
    public class MenuItem
    {
        public MenuItem(string name, int amount, double price)
        {
            Name = name;
            Amount = amount;
            Price = price;
        }

        public string Name { get; set; }

        public int Amount { get; set; }

        public double Price { get; set; }

        public void Display()
        {
            Console.WriteLine("\nName: " + Name);
            Console.WriteLine("Amount:" + Amount.ToString());
            Console.WriteLine("Price: $" + Price.ToString());
        }
    }
}

File: src/PHP/BehavioralPatterns/Command/FastFoodRestaurant/Models/MenuItem.php
<?php

namespace BehavioralPatterns\Command\FastFoodRestaurant\Models;

class MenuItem {
    public string $name;
    public float $amount;
    public float $price;

    public function __construct($name, $amount, $price) {
        $this->name = $name;
        $this->amount = $amount;
        $this->price = $price;
    }

    public function display(): void
    {
        echo "\nName: " . $this->name . "\n";
        echo "Amount: " . $this->amount . "\n";
        echo "Price: $" . $this->price . "\n";
    }
}